src/Storages/System/StorageSystemStackTrace.cpp

```diff
@@ -213,6 +215,21 @@ void StorageSystemStackTrace::fillData(MutableColumns & res_columns, ContextPtr,
             throwFromErrno("Cannot send signal with sigqueue", ErrorCodes::CANNOT_SIGQUEUE);
         }
 
+        std::filesystem::path thread_name_path = it->path();
+        thread_name_path.append("comm");
+
+        std::ostringstream thread_name;
+        if (std::filesystem::exists(thread_name_path))
+        {
+            std::ifstream comm(thread_name_path.string());
+            std::string s;
+            while (thread_name && std::getline(comm, s))
+            {
+                thread_name << s;
+            }
+            comm.close();
+        }
```

Comment:
Could you please replace `std::ifstream` and `std::ostringstream` with `ReadBufferFromFile` and `readStringUntilEOF(...)`?
(that's why Style Check failed)

Reply:
Thank you for your advice, I will change it soon.